app.js

```diff
@@ -38,6 +39,7 @@ app.use("/testimonials", testimonialsRouter);
 app.use("/categories", categoriesRouter);
 app.use("/news", newsRouter);
 app.use("/activities", activitiesRouter);
+app.use("/", contactsRouter);
```

Comment:
Index is the same as **contactsRouter**.

Reply:
fixed